Assets/Mirror/Migration/Editor/Utils.cs

```diff
@@ -25,9 +33,44 @@ public class Utils : MonoBehaviour {
 
                 // destroy UNET component
                 DestroyImmediate(unetNetworkComponent, true);
+                return true;
+            }
+
+            return false;
+        }
+
+        public static bool ReplaceNetworkIdentity(GameObject prefab) {
```

Comment:
Separate method for NetworkIdentites as they require special reliability (also, from what i've noticed, CopyProperties doesn't work so just assign those available in editor manually)